crates/lumina_shared/src/player/spaceship.rs

```diff
@@ -219,7 +304,9 @@ impl MovementStat {
     ) {
         let factor = match self.linear_acceleration.total_cmp(&target) {
             std::cmp::Ordering::Less => acceleration_factor,
-            std::cmp::Ordering::Equal => return,
+            std::cmp::Ordering::Equal => {
+                return;
+            }
```

Comment:
```suggestion
            std::cmp::Ordering::Equal => return,
```

hmm why create new line tho? is it rustfmt automatically doing it?

Reply:
Yup, this all is done by the formatter when I saved..

Reply:
Kinda weird tho, mine doesn't do that haha.